components/miro_custom_app/actions/create-board/create-board.mjs

```diff
@@ -0,0 +1,36 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  name: "Create Board",
+  version: "0.0.7",
+  key: "create-board",
+  description: "Creates a Miro board",
+  type: "action",
+  props: {
+    miro_custom_app: {
+      type: "app",
+      app: "miro_custom_app"
+    },
```

Comment:
Once you import the `app` (main file) in this component then you can make this change:
```suggestion
    app,
```
the same applies to the rest of components! `create-shape, create-sticky-note, delete-board, delete-item, get-board, get-items, get-specific-item, list-boards, update-board, update-shape, update-sticky-note`